apps/prairielearn/src/lib/course.ts

```diff
@@ -192,3 +193,31 @@ export async function pullAndUpdateCourse({
 
   return { jobSequenceId: serverJob.jobSequenceId, jobPromise };
 }
+
+/**
+ * Return the courses that the user has staff access to.
+ */
+export async function selectInstructorCourses({
+  userId,
+  isAdministrator,
+  includeExampleCourse,
+}: {
+  userId: string;
+  isAdministrator: boolean;
+  /**
+   * Example courses are only shown to users who are either instructors of
+   * at least one other course, or who are admins. They're also shown
+   * unconditionally in dev mode.
+   */
+  includeExampleCourse: boolean;
+}) {
+  return await sqldb.queryRows(
+    sql.select_instructor_courses,
+    {
+      user_id: userId,
+      is_administrator: isAdministrator,
+      include_example_course: includeExampleCourse,
+    },
+    InstructorCourseSchema,
+  );
+}
```

Comment:
Nit: AFAICT this function isn't actually used by any part of this PR other than the homepage, where the query originally lived. Would you mind reverting this change to cut down on the scope of this PR?

Reply:
Thanks!